spec/components/select_spec.rb

```diff
@@ -87,5 +95,22 @@
     end
   end
 
+  context 'with label_col and control_col options provided' do
+    let(:form) do
+      helper.bootstrap_form_for :foo, url: '/', layout: :horizontal do |f|
+        return f
```

Comment:
How does this magic actually work :smile: ?  Does it just force an early return with the form in the `bootstrap_form_for` definition?